tests/integration/editor/test_editor_views.py

```diff
@@ -287,7 +384,8 @@ def test_refextract_text(log_in_as_cataloger, api_client):
 
     assert response.status_code == 200
     assert validate(references, subschema) is None
-    assert get_value({'references': references}, 'references.reference.publication_info.journal_title')
+    assert get_value({'references': references},
+                     'references.reference.publication_info.journal_title')
```

Comment:
Check your editor! Our `pytest.ini` ignores `E501`, so your style checks should be ignoring it as well.